www/creating-zooming-images.html

```diff
@@ -121,8 +143,16 @@ <h2>Creating Zooming Images</h2>
         <td><a href="/examples/tilesource-zoomify/">Zoomify</a></td>
     </tr>
     <tr>
-        <td><a href="https://github.com/usnistgov/pyramidio">PyramidIO</a></td>
-        <td>Java (command line and library)</td>
-        <td><a href="/examples/tilesource-dzi/">DZI</a></td>
+        <td><a href="http://www.zoomify.com/photoshop.htm">Zoomify Plugin</a></td>
+        <td>Free proprietary export plugin for Photoshop</td>
+        <td><a href="/examples/tilesource-zoomify/">Zoomify</a></td>
     </tr>
 </table>
+<p>If none of these options meet your needs, you can create your own tile-generating tool. See <a href="https://github.com/openseadragon/openseadragon/wiki/The-DZI-File-Format">this page</a> for more information on the DZI format.</p>
+<h3>Hosting Zooming Images</h3>
```

Comment:
Actually I think these should be integrated with the list above. They are both for IIIF, so they can go in that section. The description can be just "server" or something.